lib/templates/components/content/CeRenderer.js

```diff
@@ -8,45 +8,44 @@ export default {
      */
     content: {
       type: Array,
-      required: true,
       default: () => []
```

Comment:
@mercs600 I think that in this case we should also use required prop instead of default. Should I add it to the latest commit?